src/Service/token-service.js

```diff
@@ -0,0 +1,47 @@
+const jwt = require('jsonwebtoken');
+const tokenModel = require('../Models/token-model');
```

Comment:
absolute path

Reply:
and why do you give the model a new name if you have it in the file with the model called "Token"?